single_stage_detector/ssd/mlcube.py

```diff
@@ -0,0 +1,99 @@
+import os
+import yaml
+import typer
+import shutil
+import subprocess
+from pathlib import Path
+
+
+app = typer.Typer()
+
+
+class DownloadDataTask(object):
+
+    urls = {
```

Comment:
seems like this entire file except these urls is boilerplate copy/paste, so there should just be one copy of the code in the central mlcube distribution.  Then these urls can go in the single_stage_detector/mlcube/.mlcube.yaml file.